### DISCUSSION TITLE
Zigbee2MQTT constantly crashing (Attempt to access memory outside buffer bounds at new NodeError)

### DISCUSSION THREAD

**Discussion by chwarg** [Koenkk/zigbee2mqtt#22805] [Devices]
Hi I recently have issues with Zigbee2MQTT constantly crashing on me.
I run it alongside with Home Assistant on a Raspberry Pi 4 with 2GB RAM.

These are the error messages I constantly get:

Error 2024-05-28 07:39:38Unparsed frame 0xc4. Skipped
Error 2024-05-28 07:39:58Frame changeSourceRouteHandler parsing error: RangeError: Attempt to access memory outside buffer bounds at new NodeError (node:internal/errors:405:5) at boundsError (node:internal/buffer:86:11) at Buffer.readUInt16LE (node:internal/buffer:245:5) at Buffer.readUIntLE (node:internal/buffer:182:17) at Function.deserialize (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/types/basic.ts:19:67) at new EZSPFrameData (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:180:54) at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:154:23 at Array.every (<anonymous>) at Function.createFrame (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:148:15) at Ezsp.onFrameReceived (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/ezsp.ts:439:35)

Any idea how I can fix that?
Thank you for your help!

**Comment by francisp2** [comment]
How is Zigbee2MQTT installed ?

**Reply by chwarg** [reply]
It was installed as an AddOn in Home Assistant.

**Comment by chwarg** [comment]
Here's info about the version.
![Screenshot_20240528_075635_Home Assistant](https://github.com/Koenkk/zigbee2mqtt/assets/73500245/549576d9-db3d-4b85-8453-1c0a0e37c31f)

**Comment by Koenkk** [comment]
You are using the old `ezsp` driver, it's recommended to use the new `ember` drivers ([info](https://github.com/Koenkk/zigbee2mqtt/discussions/21462))
